src/markdown-pages/docs/ecosystem/ecosystem-authentication.md

```diff
@@ -51,3 +51,27 @@ Total:2
 
 The returned token list is sorted in creation date order, with the earliest creation date first. The description information matches the description that is provided by the user when creating a new access token from the Web UI.
 
+
+### Revoking personal access tokens
+
+If a user moves to a new role, or loses a laptop with their personal access token on it, you can log into the Galasa Ecosystem and revoke their access token by using the `galasactl auth tokens delete` command. This ensures that the Ecosystem and systems-under-test remain secure.
+
+You can retrieve a list of available personal access tokens that have been created, along with their token IDs, by running the `galasactl auth tokens get` command, as described in the previous section. 
+
+
+Run the following command to revoke a personal access token with the token ID `myId`:
+
+```
+galasactl auth tokens delete --tokenid myId
+```
+
+*Note:* The `galasactl auth tokens delete` command revokes personal access tokens that a user creates through the WebUI. When a user runs a CLI command that talks to the Ecosystem, the CLI uses the personal access token to get a JSON Web Token (JWT). A JWT is a separate, temporary token that identifies a user and is used in galasactl commands to talk to the API server. JWTs cannot be revoked, but they do expire, so a user can continue to run CLI commands after revoking their personal access token until their JWT expires. You can remove the JWT that is stored on a user's machine instead of having to wait for the JWT to expire, by running the `galasactl auth logout` command on that machine.
```

Comment:
Minor suggestion, `a user creates through the WebUI` could be changed to `a user creates through their Ecosystem's Web UI` in case of confusion around what "the Web UI" means

Reply:
as per discussion changed to Galasa Web UI for consistency